core/src/main/resources/hudson/model/Messages_ru.properties

```diff
@@ -82,6 +85,9 @@ Job.Pronoun=Проект
 Job.minutes=мин
 Job.NoRenameWhileBuilding=Невозможно переименовать задачу при активных сборках
 
+Node.Mode.NORMAL=Загружать этот узел настолько, насколько возможно
```

Comment:
Hm, looks weird, Node.Mode.NORMAL but description as it is Node.Mode.HIGH or equal.

Reply:
the options are NORMAL and EXCLUSIVE, so I think it's "normal" as in "default", "as usual".

Reply:
thanks for explanation